### PR TITLE
Add pdf signature tutorial

### PR DESCRIPTION
**Pull request by epolevikov**

### CODE REVIEW

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Reports with a Visual PDF Signature
+author: Eugene Polevikov
+---
+# Reports with a Visual PDF Signature
+
+This tutorial describes how to create a report with a visual PDF signature.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md:7]: What is the difference between **a visual PDF signature** and **a PDF signature**?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+![](../../../../images/pdf-signature-report-result-after-sign-lower.png)
+
+## Create a Report Layout
+
+1. Drop the **RichText** control from the report controls Toolbox tab onto the **Detail** band.
+
+    ![](../../../../images/pdf-signature-report-drop-rich-text.png)
+
+2. Double-click the control and insert text of the [DevExpress Website Terms of Use](https://www.devexpress.com/aboutus/legal.xml).
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md:17]:
```suggestion
2. Double-click the control and insert the [DevExpress Website Terms of Use](https://www.devexpress.com/aboutus/legal.xml) text.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+4. Drop the **PdfSignature** control from the report controls Toolbox tab onto the **Report Footer** band.
+
+    ![](../../../../images/pdf-signature-report-add-pdf-signature.png)
+
+5. Place the **Label** control to the left of the **PdfSignature** control and add the following text: _I have read and accept this Website Terms of Use statement_.
+
+    ![](../../../../images/pdf-signature-report-add-xr-label-and-line.png)
+
+Click the **Preview Tab** to see the result.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md:33]:
```suggestion
Click the **Preview Tab** to show the result.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+2. In the invoked **PDF Export Options** dialog, click the **Digital Signature** option's ellipsis button.
+
+    ![](../../../../images/pdf-signature-report-sign-2.png)
+
+3. In the invoked **Signature Options** dialog, choose a certificate and specify signature details.
+
+    ![](../../../../images/pdf-signature-report-sign-3.png)
+
+Save and open the document to see the final result.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md:51]:
```suggestion
Save and open the document to show the final result.
```

**Review by dirk-pieterse — APPROVED** [review]

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md (hunk 1 above)
Comment (epolevikov) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/create-popular-reports/reports-with-visual-pdf-signature.md:7]: You can sign a PDF document but the signature won't be displayed in this document. You should check the document's properties to ensure the document is signed. A **visual** signature allows you to visualize a signature in a document itself.

**Pull request merged**